src/components/rockets/RocketsLists.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { reserveRocket } from '../../redux/rockets/RocketsSlice';
+import { FetchData } from '../../redux/rockets/Api';
+import styles from './rocket.module.css';
+
+const RocketList = () => {
+  const { rockets, status, error } = useSelector((state) => state.rockets);
+  const Dispatch = useDispatch();
+  console.log(rockets);
+
+  useEffect(() => {
+    Dispatch(FetchData());
+  }, [Dispatch]);
+
+  if (status) {
+    return 'Loading...';
+  }
+
+  if (error) {
+    return 'An error occurred';
+  }
+
+  return (
+    <div className="rocketContainer">
```

Comment:
- [ ] Good job creating the rockets components 👍 ,I noticed that an unusual result when it comes to this component  👍 . Whenever the Rockets page is refreshed ,  all the state is reset , Kindly ensure that once the user refresh the Rockets page , the state does not have to be reset 😄 

### Before Re-rendered 

![Capture](https://github.com/GhostEsso/SpaceTravelersHub/assets/80458199/3512e6d5-cca6-49d5-a707-87cb4dca9cd7)

### After Re-rendered 

![Capture1](https://github.com/GhostEsso/SpaceTravelersHub/assets/80458199/f2033a50-5590-4491-bbf6-09b0ff7d93da)

Noticed that the `Reserved rocket` state has changed by itself 👍  , Kindly fix that